packages/lesswrong/components/common/HeaderSubtitle.tsx

```diff
@@ -8,6 +8,7 @@ export const styles = (theme: ThemeType): JssStyles => ({
     marginLeft: '1em',
     paddingLeft: '1em',
     textTransform: 'uppercase',
+    color: theme.palette.header.text,
```

Comment:
lack-of-this was resulting in all sub-headers on AF being black, which looked bad on the dark blueish header.